### ISSUE TITLE
Create color independent toolbar differentiator between prod and staging environments 

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#9819]
## Background

Currently, the admin toolbar color is the main indicator for environments: training, staging and prod. From the Accessibility Lead [CMS audit](https://docs.google.com/spreadsheets/d/1kduP98QBltqtFG8sIB-h-vLXkEoZNPFJ14lEfT6icus/edit#gid=1653727169) this was determined to be a 508 defect level 2. To become [WCAG 2.0 criterion 1.4.1 ](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)Level A compliant, the following should be rectified:

There should be some indicator, not based on color to tell the user which environment they are working in.  There is currently text that is visually hidden that says "Staging" as well as a string of characters.  Ideally, this text would be shown without the characters so the user can clearly see they are in the staging environment.

Example of current state staging toolbar:
<img width="1256" alt="Screen Shot 2022-07-17 at 8 41 59 PM" src="https://user-images.githubusercontent.com/90111970/179434345-1670cc2e-83b4-4009-a7d0-22a67f2b824c.png">

## Acceptance Criteria

- [x] determine how to differentiate environments (training, staging, prod)
- [x] Interaction design review
- [x] Review design with Accessibility Lead
- [x] Review design with Sitewide CMS developer
- [x] Review with PM for implementation steps

**Comment by cjterdi** [comment]
Other issues to address in a future sprint:
1. Break points for tool bar: With an added label the breakpoints need to meet accessibility guidelines
2. Hover state text color dark blue does not meet AAA standard 7.1 contrast

**Comment by EWashb** [comment]
next step: create ticket for implementation
